example_configs/tomcat.json

```diff
@@ -0,0 +1,34 @@
+{
+  "rules": [
+    {
+      "pattern": "Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>requestCount:",
+      "name": "tomcat_request_count",
```

Comment:
If the java agent isn't practical for TomCat, then it'd be good to get that code into an official prometheus repo somewhere.